### PR TITLE
Issues/#308 market model restyle

### PR DESCRIPTION
**Pull request by grdw**
Fixes: #308

### CODE REVIEW

File: app/views/market_models/show.html.haml [hunk 1]
```diff
@@ -1,17 +1,16 @@
 %h1
   = @market_model.name
-%table.interactions.table
+%table.interactions.table.table-bordered
   %tr
-    %th Stakeholder from
-    %th Stakeholder To
+    %th From stakeholder
+    %th To stakeholder
```
Comment (antw) [inline app/views/market_models/show.html.haml]: @ChaelKruip @grdw Would it be more meaningful to rename "From stakeholder" and "To stakeholder" to "Payer" and "Payee" respectively? I'm struggling to think of a situation where "from" isn't paying "to", and these terms might be a bit easier to understand to novice users.

Comment (ChaelKruip) [inline app/views/market_models/show.html.haml]: I like that! :+1:

**Comment by antw** [comment]
To add to the comment about "payer" and "payee", the market model *form* also needs to be updated to remove the "Measure" column, and rename "Foundation" to "Measure".

screen shot 2015-09-01 at 12 46 12

The table on "market_models/show" looks good now.

**Comment by grdw** [comment]
screen shot 2015-09-07 at 3 42 21 pm

Like this?

**Comment by antw** [comment]
:+1:

**Pull request merged**